keras_nlp/layers/transformer_decoder.py

```diff
@@ -40,6 +41,10 @@ class TransformerDecoder(keras.layers.Layer):
         layer_norm_epsilon: float, defaults to 1e-5. The eps value in layer
             normalization components.
         name: string, defaults to None. The name of the layer.
+        kernel_initializer: tf.keras.initializers initializer, defaults to None. Sets the
+            kernel initializer for the dense and multiheaded attention layers
```

Comment:
Generally we avoid using verbs in the docstring of args. Here kernel_initializer should have docstring "The kernel initializer for the..." instead of ""Sets the kernel initializer for the...". Please change, thanks! 

Additionally, please add period ('.') to the end of docstring, thanks!